src/mobX/states/WorkpieceStates/RightsSplits.js

```diff
@@ -52,9 +52,9 @@ export default class RightsSplits {
 		// Observe splits shareholders and set hasChanged to
 		// true on first change
 		this._disposers = [
-			this.copyright.shareHolders.observe(this._toggleHasChanged),
-			// this.performance.shareHolders.observe(this._toggleHasChanged),
-			// this.recording.shareHolders.observe(this._toggleHasChanged),
+			this.copyright.shareholders.observe(this._toggleHasChanged),
```

Comment:
this line here is one that is causing problems in mobx-6

Reply:
would be good to check out the mobx-6 branch uncomment this line (i commented it to build) and see alternatives